### ISSUE TITLE
Command+Delete to eject always states that networked volumes are in use

### ISSUE THREAD

**Issue by budgetdevv** [budgetdevv/MacTweaks#9]
Find out why and resolve the issue

**Comment by budgetdevv** [comment]
Resolved by 63b586b46f0435a20cb06acc999719179cac8e64. Happened due to invalid formatting of volume paths I believe

**Comment by budgetdevv** [comment]
Apparently that wasn't the only issue. It is caused by the app running with sudo privileges.

```
trumpmcdonaldz@TrumpMcDonaldz ~ % sudo su
sh-3.2# lsof /Volumes/4TB
COMMAND PID USER   FD   TYPE DEVICE SIZE/OFF NODE NAME
mds     122 root   25r   DIR  54,14    16384    2 /Volumes/4TB
```

**Comment by budgetdevv** [comment]
Resolved by 042217e5e240ff1160b1c3a2ae70542e59ef647b

**Comment by budgetdevv** [comment]
Previous commit was a mistake. Resolved by 8f83b41967ea6624fe15490fd912d4c5bfdcf67b

**Comment by budgetdevv** [comment]
Resolved by 
91c59ec

**Comment by budgetdevv** [comment]
Resolved by 91c59ec
